### PR TITLE
Add TypeScript definition

### PR DESCRIPTION
**Pull request by Akim95**
Feel free to contribute and keep it improving!

### CODE REVIEW

**Comment by sindresorhus** [comment]
Thanks for creating a TypeScript definition. 🙌 Can you follow [this styleguide](https://github.com/sindresorhus/typescript-definition-style-guide)?

Some examples:
- https://github.com/sindresorhus/semver-regex/commit/039944bb5b3876913171b895488874b5c200e458
- https://github.com/sindresorhus/global-dirs/commit/0aae4d64887f77af94c4fb7ca12aa9f93c0a8291
- https://github.com/sindresorhus/md5-hex/commit/bbbfeff840f26973fbffc16d4863ae6d120079f3

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+// TypeScript Version: 3.6.3
+
+interface Options {
+  onlyFirst: boolean;
```
Comment (sindresorhus) [inline index.d.ts]: This option needs a doc comment. Seehttps://github.com/sindresorhus/typescript-definition-style-guide#documentation

**Comment by sindresorhus** [comment]
You need to wrap the options type in a namespace, see https://github.com/sindresorhus/typescript-definition-style-guide#documentation

File: index.test-d.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import ansiRegex from '.';
+import {expectType} from 'tsd';
+
+expectType<RegExp>(ansiRegex());
```
Comment (sindresorhus) [inline index.test-d.ts]: Add a test for the option too.

**Comment by sindresorhus** [comment]
You still need to do https://github.com/chalk/ansi-regex/pull/32#issuecomment-537818686 and use tab-indentation.

**Comment by sindresorhus** [comment]
Thanks :)

**Comment by Akim95** [comment]
Welcome! :D

**Pull request merged**